core/src/Streamly/Internal/Data/MutArray/Type.hs

```diff
@@ -2625,21 +2625,21 @@ byteEq arr1 arr2 = fmap (EQ ==) $ byteCmp arr1 arr2
 -- Compact
 -------------------------------------------------------------------------------
 
--- | Parser @pCompactChunksLE n@ coalesces adjacent arrays in the input stream
+-- | Parser @pCompactLE n@ coalesces adjacent arrays in the input stream
 -- only if the combined size would be less than or equal to n.
 --
 -- /Internal/
-{-# INLINE_NORMAL pCompactChunksLE #-}
-pCompactChunksLE ::
+{-# INLINE_NORMAL pCompactLE #-}
+pCompactLE ::
        forall m a. (MonadIO m, Unbox a)
     => Int -> Parser (MutArray a) m (MutArray a)
-pCompactChunksLE n = Parser step initial extract
+pCompactLE n = Parser step initial extract
```

Comment:
See previous comments on naming and comments on the other related PR regarding naming.